test/steps/2_when.js

```diff
@@ -44,6 +44,13 @@ import {
 
 const tablifyToSpy = require('../../src/utils/tablify');
 
+export function theActionIsCalledWithTheKeysgroupTheLifetimeTheMinimumOfSignaturesAndTheOptions() {
+	const { action, lifetime, keysgroup, minimum, options } = this.test.ctx;
+	const returnValue = action({ lifetime, keysgroup, minimum, options });
```

Comment:
These values should be converted to relevant strings before being passed to the action.